ttter/accounts/views.py

```diff
@@ -39,3 +43,81 @@ def form_valid(self, form):
 
     def form_invalid(self, form):
         return render(self.request, "accounts/create.html", {"form": form})
+
+
+class UserListView(LoginRequiredMixin, ListView):
+    template_name = "accounts/user_list.html"
+    model = User
+    queryset = User.objects.values_list("username", flat=True)
+
+
+class UserPage(LoginRequiredMixin, TemplateView):
+    template_name = "accounts/user_page.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        request_user = self.request.user
+        user = get_object_or_404(User, username=self.kwargs.get("username"))
+        post_item = user.tweet_set.all()
+        if request_user != user:
+            context["is_followed"] = FriendShip.objects.filter(
+                followee=request_user, follower=user
+            ).exists()
+        context["user"] = user
+        context["post_item"] = post_item
```

Comment:
```suggestion
        context["post_item"] = user.tweet_set.all()
```
post_itemは他で使ってなさそうなので、このままcontextとした方が見やすいです

Reply:
了解です！